sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/BridgeInternal.java

```diff
@@ -143,14 +143,16 @@ public static <T> FeedResponse<T> createFeedResponseWithQueryMetrics(
         QueryInfo.QueryPlanDiagnosticsContext diagnosticsContext,
         boolean useEtagAsContinuation,
         boolean isNoChangesResponse,
-        CosmosDiagnostics cosmosDiagnostics) {
```

Comment:
No changes to BridgeInternal (excdept deleting methods). Agreement with centralk team is that any changes we will instead use ImpelemntationBridgeHelper and delete the "old" overload in BridgeInternal. Deletion is the only chaneg they will let pass - and since this is technically public API it will show up in API design review.

Reply:
Reverted changes.